Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransforms.java

```diff
@@ -215,18 +211,17 @@ private ParticipantObjectIdentificationType createPatientParticipantObjectIdenti
     }
 
     private ParticipantObjectIdentificationType createParticipantObject(short objTypeCodeSys, short objTypeCodeRole,
-        String objIdTypeCode, String objIdTypeCodeSys, String objIdTypeDisplayName) {
+            String objIdTypeCode, String objIdTypeCodeSys, String objIdTypeDisplayName) {
 
         return createParticipantObjectIdentification(objTypeCodeSys, objTypeCodeRole,
-            objIdTypeCode, objIdTypeCodeSys, objIdTypeDisplayName);
+                objIdTypeCode, objIdTypeCodeSys, objIdTypeDisplayName);
     }
 
     private byte[] getParticipantObjectQueryForRequest(AdhocQueryRequest request) throws JAXBException {
         StringWriter sw = new StringWriter();
```

Comment:
This is not changed as part of the pr. But will address it as part of this pr so that we have a consistency across services.